LambdaRuntimeDockerfiles/SmokeTests/test/ImageFunction.SmokeTests/ImageFunctionTests.cs

```diff
@@ -170,6 +171,7 @@ private async Task CreateFunctionAsync()
                         MemorySize = 512,
                         Role = _executionRoleArn,
                         PackageType = PackageType.Image,
+                        Timeout = 30,
```

Comment:
I added the timeout because the default value is 3 seconds and apparently listing all of the timezones at least for the first time can be slow.